src/common/SPELLS_PRIEST.js

```diff
@@ -415,4 +415,121 @@ export default {
     icon: 'inv_enchant_essencemagiclarge',
   },
 
+  // Shadow Spells
+
+  VOID_TORRENT: {
+    id: 205065,
+    name: 'Void Torrent',
+    icon: 'inv_knife_1h_artifactcthun_d_01',
+  },
+
+  MIND_BLAST: {
+    id: 8092,
+    name: 'Mind Blast',
+    icon: 'spell_shadow_unholyfrenzy',
+  },
+
+  MIND_FLAY: {
+    id: 15407,
+    name: 'Mind Flay',
+    icon: 'spell_shadow_siphonmana',
+  },
+
+  VAMPIRIC_TOUCH: {
+    id: 34914,
+    name: 'Vampiric Touch',
+    icon: 'spell_holy_stoicism',
+  },
+
+  SHADOW_WORD_DEATH: {
+    id: 32379,
+    name: 'Shadow Word: Death',
+    icon: 'spell_shadow_demonicfortitude',
+  },
+
+  VOID_ERUPTION: {
+    id: 228260,
+    name: 'Void Eruption',
+    icon: 'spell_priest_void-blast',
+  },
+
+  VOID_BOLT: {
+    id: 205448,
+    name: 'Void Bolt',
+    icon: 'ability_ironmaidens_convulsiveshadows',
+  },
+
+  MINDBENDER: {
```

Comment:
This talent is available under `MINDBENDER_TALENT_SHADOW`, this should be used instead.